code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm

```diff
@@ -24,16 +24,16 @@
 				. =  FALSE
 			switch(dir)
 				if(SOUTHEAST)
-					if(object.dir != SOUTH)
+					if(object.dir != dir)
 						. = FALSE
 				if(SOUTHWEST)
-					if(object.dir != WEST)
+					if(object.dir != dir)
 						. =  FALSE
 				if(NORTHEAST)
-					if(object.dir != EAST)
+					if(object.dir != dir)
 						. =  FALSE
 				if(NORTHWEST)
-					if(object.dir != NORTH)
+					if(object.dir != dir)
 						. =  FALSE
```

Comment:
What's going on here?

Reply:
since i changed what direction the corners have, i had to change this too (they went from cardinal to diagonal direction)
this doesn't change a thing in game.. should i have mentioned it? it seemed a small change to me

Reply:
no I was just confused, didn't really understand what was going on is all